data-canary/scripts/talkactions/player/server_info.lua

```diff
@@ -1,11 +1,38 @@
 local serverInfo = TalkAction("!serverinfo")
 
 function serverInfo.onSay(player, words, param)
-	player:sendTextMessage(MESSAGE_EVENT_ADVANCE, "Server Info:"
-	.. "\nExp rate: " .. getRateFromTable(experienceStages, player:getLevel(), configManager.getNumber(configKeys.RATE_EXPERIENCE))
-	.. "\nSkill rate: " .. configManager.getNumber(configKeys.RATE_SKILL)
-	.. "\nMagic rate: " .. configManager.getNumber(configKeys.RATE_MAGIC)
-	.. "\nLoot rate: " .. configManager.getNumber(configKeys.RATE_LOOT))
+	local configRateSkill =  configManager.getNumber(configKeys.RATE_SKILL)
+	local rateUseStages =  configManager.getNumber(configKeys.RATE_USE_STAGES)
```

Comment:
Why did you create this variable?

Reply:
maintaining writing standards and ensuring smooth operation